packages/ui/components/listbox/test-suites/ListboxMixin.suite.js

```diff
@@ -673,12 +674,40 @@ export function runListboxMixinSuite(customConfig = {}) {
             mimicKeyPress(_listboxNode, 'Enter');
             expect(options[1].checked).to.be.true;
           });
+
+          it('submits form on [Enter] when inputNode is an instance of HTMLInputNode', async () => {
+            const submitSpy = sinon.spy(e => e.preventDefault());
+            const el = /** @type {HTMLFormElement}  */ (
+              await _fixture(html`
+              <form @submit=${submitSpy}>
+                <${tag} name="foo">
+                  <${optionTag} .choiceValue="${'Artichoke'}">Artichoke</${optionTag}>
+                  <${optionTag} .choiceValue="${'Bla'}">Bla</${optionTag}>
+                  <${optionTag} .choiceValue="${'Chard'}">Chard</${optionTag}>
+                </${tag}>
+                <button type="submit">submit</button>
+              </form>
+            `)
+            );
+            const listbox = /** @type {LionListbox} */ (el.querySelector('[name="foo"]'));
+            const { _inputNode } = getListboxMembers(listbox);
+            await listbox.updateComplete;
```

Comment:
```suggestion
            await listbox.updateComplete;
            if (!_inputNode instanceof HTMLInputElement) {
              return;
            }
```

Better readable and cheaper to do early return